Tests/StoreKitUnitTests/TestHelpers/AvailabilityChecks.swift

```diff
@@ -19,6 +19,12 @@ import XCTest
 // everything in those classes will still be called by XCTest, and it will cause errors.
 enum AvailabilityChecks {
 
+    static func iOS12APIAvailableOrSkipTest() throws {
```

Comment:
Also unused, but maybe you plan to use it for future tests?

Reply:
Oh not anymore, since this feature is iOS 13 only, I'll remove it!